entry/src/ohosTest/java/info/metadude/android/typedpreferences/demo/DoublePrefTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2020-21 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package info.metadude.android.typedpreferences.demo;
+
+import info.metadude.android.typedpreferences.*;
+import ohos.aafwk.ability.delegation.AbilityDelegatorRegistry;
+import ohos.agp.render.render3d.BuildConfig;
+import ohos.app.Context;
+import ohos.data.DatabaseHelper;
+import ohos.data.preferences.Preferences;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class DoublePrefTest {
+
+    private static final String PREFERENCES_KEY = BuildConfig.APPLICATION_ID+".TEST_KEY_STRING";
+    private static final String APP_RATER = "apprater";
+
+
+    private DoublePreference mpreference2;
+    private static final double MTESTVALUE2 = 23.23d;
+    private static final double MDEFAULTVALUE2 = 42.42f;
+    private static final float MDELTA2 = 0f;
+
+    @Before
+    public void setUp()  {
+        Preferences preferences;
```

Comment:
Variables - preferences, databaseHelper, context;

declaration and initialization can be done in a single line.